main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        dog
+//  action 1:    the dog sits
+dog.sit();
+//  action 2:    the dog barks
+dog.bark();
+//  action 3:    the dog wags its tail
+dog.wagTail();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        tree
+//  action 1:    the tree grows
+tree.grow();
+//  action 2:    the tree sheds leaves
+tree.shedLeaves();
+//  action 3:    the tree buds new leaves
+tree.bud();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        computer
+//  action 1:    the computer logs data
+computer.logData();
+//  action 2:    the computer deletes data
+computer.deleteData();
+//  action 3:    the computer crashes :(
+computer.crash();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        jason    
+//  action 1:    jason eats
+jason.eat();
+//  action 2:    jason sleeps 
+jason.sleep();
+//  action 3:    jason codes
+jason.code();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        cloud
+//  action 1:    the cloud drifts
+cloud.drift();
+//  action 2:    the cloud grows
+cloud.grow();
+//  action 3:    the cloud rains
+cloud.rain();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        guitar
+//  action 1:    the guitar walks
+guitar.walk();
+//  action 2:    the guitar shreds
+guitar.shred();
+//  action 3:    the guitar gently weeps
+guitar.gentlyWeep();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        rock
+//  action 1:    the rock tumbles
+rock.tumble();
+//  action 2:    the rock rolls
+rock.roll();
+//  action 3:    the rock sits motionless for eons
+rock.abide();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        bird   
+//  action 1:    the bird flies
+bird.fly();
+//  action 2:    the bird screeches 
+bird.screech();
+//  action 3:    the bird sits on a wire
+bird.sitOnWire();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.